tests/kola/test-coreos.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+set -ex
+
+REPO=https://github.com/coreos/coreos-assembler.git
```

Comment:
Please be more specific with the variable name

Reply:
Done